src/sql.c

```diff
@@ -400,6 +402,16 @@ sql_x (char* sql, va_list args, sql_stmt_t** stmt_return)
           sql_finalize (*stmt_return);
           continue;
         }
+      else if (ret == -5)
```

Comment:
```suggestion
      if (ret == -5)
```
I know it's minor but the `else` bugs me because it makes it look like an `else` is necessary. Breaks the pattern with the `if` statements above.